.gitignore

```diff
@@ -60,3 +60,13 @@ build-iPhoneSimulator/
 downzip-sw.js
 .npmrc
 polyphemus/.bundle/config
+
+*.DS_Store
+
+# For magmaR dev via RStudio.
+etna/packages/magmaR/.Rproj.user
+etna/packages/magmaR/.Rhistory
+etna/packages/magmaR/.RData
+etna/packages/magmaR/.Ruserdata
+etna/packages/magmaR*.tar.gz
+.Rproj.user
```

Comment:
Just a note that although we don't technically need this line in the .gitignore, there's a current bug in RStudio where it will continuously try to add it if we don't put the line.  Doesn't hurt us any, so why not just have it.